RecoLocalCalo/HGCalRecAlgos/interface/HGCalImagingAlgo.h

```diff
@@ -165,6 +173,10 @@ static const unsigned int maxlayer = 52;
 static const unsigned int lastLayerEE = 28;
```

Comment:
(```maxlayer```), ```lastLayerEE``` and ```lastLayerFH``` should all be available from the geometry via RecHitTools, isn't it?  
If so, better refer to it than relying on "magic numbers"
@bsunanda